Gemfile

```diff
@@ -5,5 +5,6 @@ gem "mocha"
 gem "rake"
 gem "simplecov"
 gem "codeclimate-test-reporter", "1.0.0.pre.rc2"
+gem "pry"
```

Comment:
Hm did a quick poll of some of our core services and we seem to have pry in slightly more than half:

builder, scheduler, api, cce: yes
wally, finalizer: no

Personally I'd be in favor. I see a benefit to having pry (find it useful when writing tests) and little downside. 

Curious why you think it would
> not actually be used long-term
?